services/brig/src/Brig/API.hs

```diff
@@ -1467,6 +1475,19 @@ addBlacklist emailOrPhone = do
     void . lift $ API.blacklistInsert emailOrPhone
     return empty
 
+canBeDeleted :: UserId ::: TeamId -> Handler Response
+canBeDeleted (uid ::: tid) = do
+    onlyOwner <- lift (Team.isOnlyTeamOwner uid tid)
+    case onlyOwner of
+       Team.IsOnlyTeamOwner       -> throwStd noOtherOwner
+       Team.IsOneOfManyTeamOwners -> pure ()
+       Team.IsNotTeamOwner        -> pure ()
+       Team.NoTeamOwnersAreLeft   -> do
+           Log.warn $ Log.field "user" (toByteString uid)
+                    . Log.msg (Log.val "Team.NoTeamOwnersAreLeft")
+           pure ()
```

Comment:
Wasn't this removed at some point?

Reply:
i think this may be another one.  (-:  i'll remove that, too.